end-to-end-tests/cypress/support/commands.js

```diff
@@ -25,33 +25,37 @@
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
 import "@testing-library/cypress/add-commands";
 
-Cypress.Commands.add(
-  "navToViewer",
-  (viewerLinkText, navigationProfile) => {
-    let requestsMade = [];
-    for (const route of ["catalog", "system-security-plans", "component-definitions", "profiles"]) {
-      cy.intercept('GET', `${Cypress.env("api_url")}/${route}`, () => {
-        requestsMade.push(route);
-      }).as(route);
-    }
-    cy.visit(Cypress.env("base_url"));
-    cy.get("button").first().click();
+Cypress.Commands.add("navToViewer", (viewerLinkText, navigationProfile) => {
```

Comment:
Would `navToViewer` be the best name for this command? Isn't this in the `Editor`? The same goes for a lot of these commands, it is a bit unclear what is being done based on the name.

Reply:
Great question! `navToViewer` is not the best name. It looks like it was introduced in #83 where we had a "nav to rest mode" command as well... Because we sorta changed the default assumptions, you're right, we should fix the name too. Even if it means making larger-scale changes